desktop/src/splash.html

```diff
@@ -5,7 +5,7 @@
       html {
         width: 100%;
         height: 100%;
-        background: url('../../public/img/media.png') center center no-repeat;
+        background: url('../../public/img/homescreen-icon-144x144.png') center center no-repeat;
```

Comment:
Why are we showing the icon there instead of the full "Spectrum"? Not sure how that's better?